opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -515,8 +516,11 @@ export const storeLoadByIdWithRefs = async (context, user, id, opts = {}) => {
   const elements = await storeLoadByIdsWithRefs(context, user, [id], opts);
   return elements.length > 0 ? R.head(elements) : null;
 };
-export const stixLoadById = async (context, user, id, opts = {}) => {
+export const stixLoadById = async (context, user, id, opts = {}, stixVersion = StixVersion.Stix_2_1) => {
   const instance = await storeLoadByIdWithRefs(context, user, id, opts);
+  if (stixVersion === StixVersion.Stix_2_0) {
+    return instance ? convertStoreToStix_2_0(instance) : undefined;
+  }
   return instance ? convertStoreToStix(instance) : undefined;
```

Comment:
Maybe convertStoreToStix should have the version and so an unified method for conversion to avoid duplicate code of conversion